components/modal/modal.css

```diff
@@ -0,0 +1,62 @@
+.modal-container{
+    margin-top: 1.87rem;
+    display: flex;
+    flex-direction: row;
```

Comment:
no need to mention. by default it is row

Reply:
I will remove that . Thanks for pointing that out .